docs/use-service.md

```diff
@@ -17,68 +17,85 @@ Once you have your Mapzen API key you'll need include it with Terrain Tile reque
 Request a single tile with this URL pattern to get started:
 
 ```
-https://tile.mapzen.com/mapzen/terrain/v1/{format}/{z}/{x}/{y}.{extension}?api_key=your-mapzen-api-key
+https://tile.mapzen.com/mapzen/terrain/v2/{format}/{z}/{x}/{y}.{extension}?api_key=your-mapzen-api-key
```

Comment:
Technically "v2" as we've been calling it doesn't have breaking changes so I advocate keeping this as `v1` publicly on our Mapzen Terrain Tiles endpoint. Discuss!